tests/third_party/cupy/misc_tests/test_byte_bounds.py

```diff
@@ -0,0 +1,75 @@
+import dpnp

```

Comment:
please follow a common approach to third party tests:
```suggestion
import dpnp as cupy

```

Reply:
Fixed